app/pages/analyst/gis.tsx

```diff
@@ -0,0 +1,198 @@
+import { MutableRefObject, useRef, useState } from 'react';
+import { usePreloadedQuery } from 'react-relay/hooks';
+import { withRelay, RelayProps } from 'relay-nextjs';
+import { graphql } from 'react-relay';
+import styled from 'styled-components';
+import path from 'path';
+import defaultRelayOptions from 'lib/relay/withRelayOptions';
+import { DashboardTabs } from 'components/AnalystDashboard';
+import { Button } from '@button-inc/bcgov-theme';
+
+import { ButtonLink, Layout } from 'components';
+import { gisUploadedJsonQuery } from '__generated__/gisUploadedJsonQuery.graphql';
+
+const getUploadedJsonQuery = graphql`
+  query gisUploadedJsonQuery {
+    session {
+      sub
+      ...DashboardTabs_query
+    }
+  }
+`;
+
+const StyledContainer = styled.div`
+  width: 100%;
+`;
+const StyledError = styled('div')`
+  color: #e71f1f;
+  margin-top: 10px;
+`;
+
+const StyledFileDiv = styled('div')`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 1.2em;
+  border: 1px solid #c0c0c0;
+
+  margin-top: 10px;
+  & svg {
+    margin: 0 8px;
+  }
+`;
+
+const StyledButton = styled(Button)`
+  min-width: 160px;
+  white-space: nowrap;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-left: 50%;
+`;
+
+const StyledBtnContainer = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+  margin-bottom: 1.25rem;
+  margin-top: 2rem;
+  flex-direction: row;
+  justify-content: left;
+`;
+
+const UploadError = ({ error }) => {
+  if (error === 'uploadFailed') {
+    return <StyledError>File failed to upload, please try again</StyledError>;
+  }
+
+  if (error === 'fileType') {
+    return (
+      <StyledError>
+        Please use an accepted file type. Accepted type for this field is: .json
+      </StyledError>
+    );
+  }
+
+  return null;
+};
+
+const GisTab = () => {
+  const hiddenFileInput = useRef() as MutableRefObject<HTMLInputElement>;
+  const [selectedFile, setSelectedFile] = useState(); 
+  const [error, setError] = useState('');
+  const acceptedFileTypes = ['.json'];
+
+  const checkFileType = (file, fileTypes) => {
+    const extension = path.extname(file)?.toLowerCase();
+
+    return fileTypes.includes(extension);
+  };
+
+  const validateFile = (file: globalThis.File) => {
```

Comment:
Same with this: can probably move this out of the component, will stop the linter from complaining